src/GeneralStateTestsFiller/stTransactionTest/CreateTransactionEOF1ValidOpcodesFiller.yml

```diff
@@ -0,0 +1,1748 @@
+# Creation transaction deploying EOF code containing each valid Opcode
+---
+CreateTransactionEOF1ValidOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      - ':label 00_stop            :raw 0x68ef000101000200000060005260096017f3'
+      - ':label 01_add             :raw 0x68ef000101000200010060005260096017f3'
```

Comment:
Many of these will be invalid with EIP-5450 because of stack underflow, but I think it's fine to leave it like this for now.

Reply:
There are also more test cases for EIP-3670 which also are pushing to the stack, but the stack is not emptied at the end... so this would also be invalid in EIP-5450. I'm leaving this as it for now (as it is still valid in EIP-3670).